backend/main/server/models.py

```diff
@@ -121,3 +121,64 @@ class VideoSchema(ma.Schema):
     artistLink = fields.String(required=False)
     username = fields.String(required=True)
     title = fields.String(required=False)
+
+
+# TODO add Message + translated message
```

Comment:
I don't think we are going to be translating tweets anytime soon

Reply:
No, but we're going to need this for WWS's messages.

Reply:
ah that's true, I forgot. LGTM